src/main/scala/mesosphere/marathon/api/akkahttp/v2/AppsController.scala

```diff
@@ -144,7 +145,7 @@ class AppsController(
         case None =>
           reject(Rejections.EntityNotFound.noApp(appId))
         case Some(info) =>
-          authorized(ViewResource, info.app).apply {
+          authorized(ViewResource, SystemConfig).apply {
```

Comment:
This is not tied to the app... this is permission of `/info` which is part of SystemConfig.

Reply:
I think this should be "authorized(ViewRunSpec, info.app)" - I think it is tight to an app.

Correct me if I am wrong...